internal/ansible/controller/status/types.go

```
@@ -77,7 +77,10 @@ func NewAnsibleResultFromMap(sm map[string]interface{}) *AnsibleResult {
 		a.Failures = int(v.(int64))
 	}
 	if v, ok := sm["completion"]; ok {
-		s := v.(string)
+		s, ok := v.(string)
+		if !ok {
+			log.V(0).Info("Unexpected non-string value for `completion` key")
+		}
```

This makes sense. Probably shouldn't ignore errors.